tests/Mutator/Util/MutatorsGeneratorTest.php

```diff
@@ -13,17 +13,29 @@
 use Infection\Mutator\Arithmetic\Plus;
 use Infection\Mutator\Boolean\FalseValue;
 use Infection\Mutator\Boolean\TrueValue;
+use Infection\Mutator\Util\MutatorProfile;
 use Infection\Mutator\Util\MutatorsGenerator;
 use PHPUnit\Framework\TestCase;
 
 class MutatorsGeneratorTest extends TestCase
 {
+    private $countDefaultMutators;
+
+    protected function setUp()
+    {
+        $this->countDefaultMutators = 0;
```

Comment:
from what I see the list doesn't change between each test so either you should initialize the value only when `$countDefaultMutators === null` or do that in the `beforeClass` something method which is called only once

Reply:
Indeed it doesn't. Done.